service/deploy.py

```diff
@@ -158,10 +158,22 @@ def configure_ansible(logger):
 
 
 def build_host_name(ip):
+    """
+    Return the host name
+    * iPlant or raw
```

Comment:
Okay. Just wasn't sure if that method body was missing an if-expression or something.